exercise-13/src/com/nayema/Point.java

```diff
@@ -0,0 +1,10 @@
+package com.nayema;
+
+public class Point {
+    double x, y;
```

Comment:
@nayema for now keep the variables separated out into individual lines. Also use the `private` instance variable access modifier when you are allowing them to be only set through the constructor like on line 6.